### DISCUSSION TITLE
Still issues solving CF captchas?

### DISCUSSION THREAD

**Discussion by jacobprice808** [FlareSolverr/FlareSolverr#1099] [General]
I remember there being issues solving cloudflare tick box captchas like 6 months ago and I thought they were patched. I can't find the issue now (I think it was deleted). Lately I've been getting served a bunch of captchas because my proxies are low quality and FlareSolverr is unable to get past them. I have observed this behavior running FlareSolverr on my mac and ubuntu server with the same issues. I noticed that the [respective issue on undetected chromedriver](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/1388) is still open and there doesn't seem to be a solid resolution. I haven't seen any discussion of it on this side, however. Are you guys still having issues with CF captchas? If so, did any of the workarounds help?

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/issues/1036
